azure_functions_worker/bindings/meta.py

```diff
@@ -94,5 +104,39 @@ def to_outgoing_proto(binding: str, obj: typing.Any, *,
             f'unable to encode outgoing TypedData: '
             f'unsupported type "{binding}" for '
             f'Python type "{type(obj).__name__}"')
+    return datum
+
```

Comment:
Is the code style in the rest of the file to add two breaklines?

Reply:
For some reason, in this file it is :/